docs/src/pages/index.tsx

```diff
@@ -4,7 +4,25 @@
  */
 
 import Layout from "@theme/Layout";
-import React from "react";
+import React, { Suspense } from "react";
+import { v4 as uuidv4 } from "uuid";
+
+import appInsights from "../appInsights";
+import CookieBanner from "../components/cookieBanner";
+
+if (typeof window !== "undefined") {
+	const userId = localStorage.getItem("userId") ?? uuidv4();
+	localStorage.setItem("userId", userId);
+	appInsights.context.user.id = userId;
+
+	const currentUrl = window.location.pathname;
+
+	let pageRoute = currentUrl;
+	pageRoute = currentUrl ?? window.location.href;
+	appInsights.trackPageView({ name: pageRoute });
+}
+
+appInsights.trackEvent({ name: "UserActivity", properties: { action: "Viewed Page" } });
```

Comment:
I find it a bit strange that this and `trackPageView()` only show up in this page... if every other page/click is tracked automatically by app insights, do we need these here?

Reply:
Yep you're right, page views are automatically tracked so calling trackPageView() is redundant. Will clean up